tests/syntax/test_errors.py

```diff
@@ -307,17 +308,16 @@ def runFile(path):
 
 @pytest.mark.parametrize('bug', (
     # BUGGY CODE                    ERROR CAUGHT DURING:
-    'x = 3 << 2',                   # token translation
```

Comment:
If there are still Python token types disallowed in Scenic, let's continue to have a test case like this one just in case such errors ever get caught differently than other syntax errors (you can change the comment of course).

Reply:
@dfremont Not sure if I understand your right. My understanding is that `<<` and other infix operators were disallowed in Scenic because they were used for substitution, but with the new parser, they can be used as plain Python operators. Are there tokens that should not be allowed? I'm not aware of one.

Reply:
You're right, there are no tokens still disallowed, so this point is moot.